src/components/draw/DrawDirective.js

```diff
@@ -34,8 +34,10 @@ goog.require('ga_permalink');
         $document, gaMeasure) {
 
       var createDefaultLayer = function(map, useTemporaryLayer) {
+        // #2820: we set useSpatialIndex to false to allow display of azimuth
+        // circle created by style.
         var dfltLayer = new ol.layer.Vector({
-          source: new ol.source.Vector(),
+          source: new ol.source.Vector({useSpatialIndex: false}),
```

Comment:
That means we draw the whole layer in each frame, independant of zoom or location, right?